### ISSUE TITLE
darkflow-yolov2 Cannot decrease moving ave loss

### ISSUE THREAD

**Issue by TheOddMan** [thtrieu/darkflow#1071]
I use yolov2.cfg(not tiny) without .weights and COCO dataset(about 64,000 images).
My yolo model only predicts "person" object.

At first, I only train 5 images from COCO trying to overfit and the prediction(bounding box) displays correctly.
Then, I train the whole dataset(about 64,000 images).
The moving ave loss drops quickly at first, but when the loss reaches about 8, it stucks.
I've tried to adjust learning rate(from 0.0001 to 1e-10 even 1e-20), but it seems not working(the loss still stucks at about 8.)

Is there any method to solve this problem?
